### PR TITLE
Ci tests

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Adding GitHub action to build and push images when new push to master or ci-tests

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Docker Image CI
+on:
+  push:
+    branch: ['master', 'ci-tests']
```
Comment (chaitanyaenr) [inline .github/workflows/docker-image.yml:4]: Maybe just build the image on code push to the master? Thoughts?

Comment (paigerube14) [reply .github/workflows/docker-image.yml:4]: Yes need to change, needed to be able to test. Trying to add in the call to kraken-hub now. Sorry about the preemptive PR opening

Comment (chaitanyaenr) [reply .github/workflows/docker-image.yml:4]: No problem at all.

Comment (paigerube14) [reply .github/workflows/docker-image.yml]: Should be goo to review now. Need the kraken-hub PR: https://github.com/cloud-bulldozer/kraken-hub/pull/22/files to go in first to be able to correctly build this one

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**